packages/flutter/lib/src/widgets/form.dart

```diff
@@ -510,6 +510,10 @@ class FormField<T> extends StatefulWidget {
   ///
   /// This is called `value` in the [DropdownButtonFormField] constructor to be
   /// consistent with [DropdownButton].
+  ///
+  /// The `initialValue` affects the form field's state in two cases:
+  /// 1. When the form field is first built, `initialValue` determines the field's initial state.
+  /// 2. When [FormField.reset] is called, the form field is reset to this `initialValue`.
```

Comment:
Ah this should be [FormState.reset]. That's why the docs test is failing.

Reply:
Yes—or I guess more specifically, `[FormFieldState.reset]` is the direct way to do it; `FormState.reset()` calls this method for all of the descendant fields.

```suggestion
  /// 2. When [FormFieldState.reset] is called (either directly or by calling
  ///    [FormState.reset]), the form field is reset to this `initialValue`.
```

Reply:
You're right, good catch!